### DISCUSSION TITLE
Navigation menu with submenus and current page indicator

### DISCUSSION THREAD

**Discussion by JonnyTech** [webstudio-is/webstudio-community#166] [Q&A]
Hello Webstudio community. I have just discovered Webstudio and am not sure if this is the best place to ask noob questions.

Currently experimenting with trying to make a small website with a simple navigation menu.

I have some main pages (page1, page2, page3, page4) with some subpages (page2a, page2b, page3a, page3b).

How can I create a menu that when I hover over page2 then displays page2a and page2b as additional links?

Then when I navigate to a page, how can I indicate (highlight / underline / emphasise) that in the nav menu?

Is there detailed documentation or a manual to study, perhaps?

Thanks in advance for any replies.

**Comment by kof** [comment]
Hi https://docs.webstudio.is/university/core-components/link#how-to-style-the-current-page-state

**Reply by kof** [reply]
Definitely start with reading docs and watching content https://www.youtube.com/playlist?list=PL4vVqpngzeT4sDlanyPe99dYl8BgUYCac

**Reply by JonnyTech** [reply]
Thank you very much @kof for your prompt reply. I had already read the article about links but had not been able to implement the current page feature due to my lack of understanding and no clear example. The whole styles / tokens feature is still confusing me.

As for the videos, I have watched all that I could find, multiple times, but my brain does not learn that way, hence looking for documentation or examples to play with.

Still also trying to figure out the submenu links, hopefully my understanding shall increase.

Marvelous product, just a very steep learning curve.
